experiments/chaos/openebs_target_network_delay/test_prerequisites.yml

```diff
@@ -67,6 +67,20 @@
       stg_engine: "{{ openebs_stg_engine.stdout }}"
   when: stg_prov == "cstor.csi.openebs.io"
 
+- block:
+  - name: Derive PV name from PVC to query storage engine type (openebs)
```

Comment:
@payes  here also is this task is needed?

Reply:
removed